tests/github/interactive_api_director/experiments/pytorch_kvasir_unet/envoy/envoy_config.yaml

```diff
@@ -1,11 +1,11 @@
 params:
-  cuda_devices: []
+  cuda_devices: [ ]
 
-optional_plugin_components: {}
+optional_plugin_components: { }
 
 shard_descriptor:
   template: kvasir_shard_descriptor.KvasirShardDescriptor
   params:
     data_folder: kvasir_data
     rank_worldsize: 1,90
-    enforce_image_hw: [300, 400]
\ No newline at end of file
+    enforce_image_hw: '300,400'
```

Comment:
Minor request. Can this be changed back to:

``` enforce_image_hw: [300,400] ``` 

and then make the director config consistent with this? It's a bit confusing to have ```[ str , str , str ]``` in one config and ```int,int``` in another

Reply:
It was changed because in original tutorial it looks like:
```yaml
enforce_image_hw: '300,400'
```
https://github.com/intel/openfl/blob/develop/openfl-tutorials/interactive_api/PyTorch_Kvasir_UNet/envoy/envoy_config.yaml